docs/getting-started/welcome.md

```diff
@@ -0,0 +1,100 @@
+---
+sidebar_label: '👉 Start Here'
+description: 'Start here as a developer to get the basics and start coding fast.'
+---
+
+import Box from '@site/src/components/Box';
+import CallToActionButton from '@site/src/components/CallToActionButton';
+
+import PartnersIcon from '../../static/img/icons/icon-partners-gradient.png';
+
+import styles from '../../src/pages/index.module.scss';
+
+# Getting Started on LUKSO
+
+## Welcome to LUKSO!
+
+The purpose of this section is to get you to coding as fast as a possible. We will therefore try to take you step by step through the basics and as fast as possible to coding.
```

Comment:
Suggest removing this sentence as it is already mentioned in the first sentence:

```suggestion
The purpose of this section is to get you to coding as fast as a possible. We will therefore try to take you step by step through the basics.
```